docs/pages/hooks/useBalance.mdx

```diff
@@ -0,0 +1,206 @@
+---
+layout: docs
+---
+
+import React from "react";
+import { useBalance } from "@scaffold-ui/hooks";
+import { DocsProvider } from "../../components/DocsProvider";
+import { mainnet } from "viem/chains";
+
+# useBalance Hook
+
+The `useBalance` hook fetches and watches an address balance on a given chain, pairing live USD conversion with a UI-friendly toggle between native currency and USD modes.
```

Comment:
I think we don't mention ui/toggle part here. User can use toggle or show both values or do whatever he wants with return values in his component